### PR TITLE
fix - Exit with error code 1 if there are errors

### PR DESCRIPTION
**Pull request by bebraw**
This is needed for CIs to fail correctly. I also pushed constants to a separate place. Ideally logging should be pluggable (a module of its own). This way you could replace it with JSON etc. mechanism (good for testing).

Let me know if you want a separate PR for pluggable logging.

### CODE REVIEW

**Comment by bebraw** [comment]
I also made an important fix related to the npm distribution. It wasn't picking up some files so I set up a source directory where to point.

File: src/index.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ const chalk = require('chalk');
 const glob = require('glob');
 const flatten = require('lodash/flatten');
 const worker = require('./worker');
+const constants = require('./constants');
```
Comment (sapegin) [inline src/index.js:9]: We can destructure here, only Node 6+ is supported ;-)

**Review by sapegin — COMMENTED** [review]
Awesome, thanks!

File: src/index.js (hunk 1 above)
Comment (sapegin) [reply src/index.js:9]: But I’ll merge it anyway, we can tweak that later.

**Comment by sapegin** [comment]
> I also made an important fix related to the npm distribution. It wasn't picking up some files so I set up a source directory where to point.

Yeah, because `worker.js` wasn’t in the `files` ;-)

File: src/index.js (hunk 1 above)
Comment (bebraw) [reply src/index.js:9]: Ok, maybe easier to grep like this so change as you like.

**Pull request merged**